crates/bevy_render/src/render_resource/pipeline.rs

```diff
@@ -87,13 +87,29 @@ impl Deref for ComputePipeline {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Default)]
+pub struct BindGroupLayoutDescriptor {
+    /// Debug label of the bind group layout descriptor. This will show up in graphics debuggers for easy identification.
+    pub label: Option<Cow<'static, str>>,
```

Comment:
as an aside, I kinda feel like we should make labels non-optional in all our code but that's for another PR.

Reply:
Yeah this would be a good follow-up.

Reply:
We should be using the same DebugName strategy that ECS uses for system names.